Source/WebKit/WebProcess/WebCoreSupport/WebBroadcastChannelRegistry.cpp

```diff
@@ -62,7 +62,7 @@ void WebBroadcastChannelRegistry::registerChannel(const WebCore::PartitionedSecu
 
     if (channelsForName.size() == 1) {
         if (auto clientOrigin = toClientOrigin(origin))
-            networkProcessConnection().send(Messages::NetworkBroadcastChannelRegistry::RegisterChannel { *clientOrigin, name }, 0);
+            Ref { networkProcessConnection() }->send(Messages::NetworkBroadcastChannelRegistry::RegisterChannel { *clientOrigin, name }, 0);
```

Comment:
nit: It would be nicer to introduce a `protectedNetworkProtectionConnection()` which returns a `Ref`, then use it here and below.